dashboard/app/views/levels/editors/fields/_aitutor_fields.html.haml

```diff
@@ -1,5 +1,10 @@
 %h1.control-legend{data: {toggle: "collapse", target: "#ai_tutor"}}
   AI Tutor
 #ai_tutor.in.collapse
-  .field
-    = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :ai_tutor_available, description: "AI Tutor Available"}
+  
+  = f.label :ai_tutor_level_specific_prompt, 'Level-Specific Prompt'
+  %p Prompt used to specify the behavior of AI Tutor on a per level basis.
+  = f.text_area :ai_tutor_level_specific_prompt, placeholder: 'Write your level-specific prompt here. If left blank, this level will use the default system prompt.', rows: 4, class:"input-block-level"
+
+  = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :ai_tutor_available, description: "Make AI Tutor Available"}
+  
```

Comment:
What do you think about switching the order so the availability checkbox is first and the input box is conditionally shown if the AI Tutor is available? Is there a scenario where a Levelbuilder would want to add a custom prompt for a level where AI Tutor is unavailable?